bittensor_cli/src/commands/wallets.py

```diff
@@ -514,15 +519,19 @@ async def overview(
     with console.status(
         f":satellite: Synchronizing with chain [white]{subtensor.network}[/white]",
         spinner="aesthetic",
-    ):
+    ) as status:
         # We are printing for every coldkey.
+        print_verbose("Fetching total balance for hotkeys", status)
         block_hash = await subtensor.substrate.get_chain_head()
         all_hotkeys, total_balance = await _get_total_balance(
             total_balance, subtensor, wallet, all_wallets
         )
 
         # We are printing for a select number of hotkeys from all_hotkeys.
         if include_hotkeys:
+            print_verbose(
+                "Fetching for select hotkeys passed in 'include_hotkeys'", status
```

Comment:
Does this work?

Reply:
Yes! 
![image](https://github.com/user-attachments/assets/918570f7-19b6-4b52-bba9-7a8e7001ff54)